assignments.Rmd

```diff
@@ -34,9 +34,9 @@ individual assignments are not guaranteed to be up to date until Monday
 the deadlines below, e.g. the first assignment is not guaranteed to be up to
 date until 7.9 at 8am).
 
-Report all results in a single, anonymous *.pdf -file and submit it in
-[peergrade](https://www.peergrade.io). Submitting empty pdf or almost
-empty pdf is not allowed. Include also any source code to the report
+Report all results in a single, **anonymous** *.pdf -file and submit it in
+[peergrade](https://www.peergrade.io). Submitting empty or almost
+empty pdf files is not allowed. Include also any source code to the report
```

Comment:
"file is" or "files are"

Reply:
I think the current form is correct. Submitting ... is not allowed, so it does not matter whether ... is plural or not

Reply:
ah, I was reading too quickly just the changed line and the whole sentence